pkg/tuf/common.go

```diff
@@ -11,14 +11,15 @@ import (
 	"path/filepath"
```

Comment:
Re - the comment at the top of this file.
The comment was relevant when the file contained transport-related code, which was indeed adapted from the Notary repo.

That is not the case anymore, so we should update the comments accordingly.

Reply:
It might be true elsewhere, so how to best resolve this overall?